gutils/unicodelibinfo.c

```diff
@@ -133,6 +140,7 @@ char *unicode_name(int32 unienc) {
      * revisit later.
      */
     if( ( unienc >= 0xAC00 && unienc <= 0xD7A3 ) && ( name_data == NULL ) ) {
+	/* replaced code below to reduce library dependencies
```

Comment:
Can we please not do this, we already are using glib so just use it.

Reply:
I'm ok with it in the executables, since you need it for the GUI anyways, but if we have an opportunity to reduce dependencies in the libraries, they can be a little more portable.
I'd like to reduce dependencies on the nox if it can be done.

Reply:
This code depends on `xasprintf`, which comes from gnulib, and is required whether or not you build with the gui.

Even if you wanted to avoid that dependency, the way you have rewritten it is not good. Instead of adding yet more magic arrays, with memcpys with magic indexing (which is hard for me and others reading this to verify that it's correct), you could have used some form of `malloc`+`sprintf`, or `asprintf`, instead of making it this unreadable and relying on yet more magic arrays

Anyway, just revert it back to how it was before, it compiled fine with or without the gui.